b2b-sample/guardio-insurance-sample-application/components/sections/sections/settingsSection/roleManagementSection/roleManagementSectionComponent.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import PeoplesIcon from "@rsuite/icons/Peoples";
+import React, { useCallback, useEffect, useState } from "react";
+import { Container } from "rsuite";
+import CreateRoleButton from "./otherComponents/createRoleButton";
+import RolesList from "./otherComponents/rolesList";
+import decodeListAllRoles from "../../../../../util/apiDecode/settings/role/decodeListAllRoles";
+import EmptySettings from "../../../../common/emptySettings";
+import SettingsTitle from "../../../../common/settingsTitle";
+
+/**
+ * 
+ * @param prop - session
+ * 
+ * @returns The role management interface section.
+ */
+export default function RoleManagementSectionComponent(prop) {
+
+    const { session } = prop;
+
+    const [ rolesList, setRolesList ] = useState([]);
+
+    useEffect(() => {
+        fetchAllRoles();
+    }, [ fetchAllRoles ]);
+
+    const fetchAllRoles = useCallback(async () => {
+
+        const res = await decodeListAllRoles(session);
+
+        if (res) {
+            setRolesList(res);
+        } else {
+            setRolesList([]);
+        }
+
+    }, [ session ]);
+
+    // const onAddIdentityProviderClick = () => {
```

Comment:
why these are commented?

Reply:
Fixed when the functionality moved to a different location 
https://github.com/Achintha444/samples-is/commit/8288cbd899e674b70a91604fa51d6d65a520c2b2#diff-fc895b6281bfab3b91148d2efca61917ce6be14868c2de023069cf5438f42c7b